src/MOI_wrapper.jl

```diff
@@ -596,31 +598,35 @@ function backward_conic!(model::Optimizer, dA::Array{Float64,2}, db::Array{Float
         z = (x, y - s, [1.0])
         u, v, w = z
 
-        Q = [
+        # reshaping vectors to matrix
+        b = b[:, :]
+        c = c[:, :]
```

Comment:
why do we need these two lines?

Reply:
reindex/convert SparseVector to SparseMatrix. turns out SparseVector's give error on multiplication